test/utils/StripCharUtilSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import uk.gov.hmrc.play.test.UnitSpec
+import utils.StripCharUtil.stripAll
+
+class StripCharUtilSpec extends UnitSpec {
```

Comment:
One likely scenario that would be good to add a test for is a user entering a full stop as pence, e.g. `£10.56.`

Reply:
Can you add tests to prove removal of leading/trailing whitespace.

Reply:
Can you add in a unit test for a standard value that doesn't need any stripping e.g. 100.56